config/config_mgmt.py

```diff
@@ -77,7 +76,7 @@ def logInFile(self, header="", obj=None, json=False):
                 df.write('\n\n{}: {}\n'.format(time, header))
                 if json:
                     dump(obj, df, indent=4)
-                else:
+                elif obj:
                     #print(obj)
                     df.write('{}: {}'.format(time, obj))
```

Comment:
else:  statement to handle other cases?

Reply:
Else is do nothing, in this case. So I left else case.